src/main/clm_varpar.F90

```diff
@@ -64,9 +64,9 @@ module clm_varpar
   ! constants for decomposition cascade
 
   integer, public, parameter :: i_met_lit  = 1
```

Comment:
We will likely rename `i_met_lit` to `i_litr1` to accommodate the loops over litter pools discussed [here.](https://github.com/ESCOMP/CTSM/pull/1340/files#r613643764)